src/modules/did-contact/did.contact.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsArray, IsString } from 'class-validator';
+
+export class DIDContactDTO {
+  @IsString()
+  @ApiProperty()
+  did: string;
+
+  @IsString()
+  @ApiProperty()
+  label: string;
+}
+
+export class RequestUserDTO {
+  @IsString()
+  @ApiProperty()
+  did: string;
+
+  @IsArray({ each: true })
+  @ApiProperty()
+  verifiedRole: verifiedRoleDTO[];
+}
+
+export class verifiedRoleDTO {
+  @IsString()
+  @ApiProperty()
+  name: string;
+
+  @IsString()
+  @ApiProperty()
+  namespace: string;
+}
```

Comment:
Good idea creating these DTOs. I wonder if they would be better placed in the `auth` module though as then see to be associated with the `req.user`?

Reply:
makes sense. will update it.